integration/astro-inspection-chamber.js.ts

```diff
@@ -1,6 +1,4 @@
-import { readFileSync } from 'node:fs';
-
-const inspectionChamber = readFileSync('node_modules/@vtbag/inspection-chamber/lib/index.js');
+import inspectionChamber from '@vtbag/inspection-chamber/lib/index.js?raw';
```

Comment:
```suggestion
import "astro/client";
import inspectionChamber from "@vtbag/inspection-chamber?raw";
```

Reply:
Yes! That's great.

Out of curiosity: when I made the change I noticed that the import statement had an error, stating that it couldn't find the module (even though it worked fine). Importing `astro/client` seems to get rid of this and the module can be found. Do you know why that is the case here?

Reply:
There is a lot stuff in that file, but we need just those 4 lines:
```
declare module '*?raw' {
	const src: string;
	export default src;
}
```

Reply:
However, this only seems to have mask the error message you have seen. In fact, the .d.ts declarations were also missing in @vtbag/inspection-chamber (Thus it was not recognized as a module). They will come with the next release.